MuxExoPlayer/src/main/java/com/mux/stats/sdk/muxstats/MuxBaseExoPlayer.java

```diff
@@ -829,6 +832,7 @@ protected BandwidthMetricData onLoad(long mediaStartTimeMs, long mediaEndTimeMs,
       segmentData.setRequestUrl(segmentUrl);
       switch (dataType) {
         case C.DATA_TYPE_MANIFEST:
+          detectMimeType = false;
```

Comment:
Is this the only case where we need to not detect the mime type? DASH will have init fragments, but it might have a manifest event first. Did you test this with a DASH source?

Reply:
This is the only place.

I tested with the dash it is good, C.DATA_TYPE_MANIFEST refer to dash main manifest (mpd file), C.DATA_TYPE_MEDIA_INITIALIZATION reffer to the first mp4 segment.

Reply:
Okay sounds good to me!